tensor2tensor/layers/modalities.py

```diff
@@ -100,7 +100,6 @@ def bottom_simple(self, x, name, reuse):
         x = tf.squeeze(x, axis=3)
       while len(x.get_shape()) < 3:
         x = tf.expand_dims(x, axis=-1)
-
```

Comment:
This doesn't seem to be necessary, can you revert and not change this file at all (just to reduce the size of this large PR).

Reply:
Sure, sorry for this.